tests/ceph_ansible/test_ansible_upgrade.py

```diff
@@ -40,7 +40,9 @@ def run(ceph_cluster, **kw):
     ceph_cluster.use_cdn = config.get("use_cdn")
 
     config["ansi_config"].update(
-        set_container_info(ceph_cluster, config, ceph_cluster.use_cdn, containerized)
+        set_container_info(
```

Comment:
As discussed, Let's consider the updating the ansible configuration for switch-non-containerized-to-containerized playbook in the respective test module.